source/codegen/templates/proto_helpers.mako

```diff
@@ -30,16 +30,22 @@ enum ${service_class_prefix}Attributes {
 <%
   enum = enums[enum_name]
   enum_value_prefix = enum.get("enum-value-prefix", common_helpers.pascal_to_snake(enum_name).upper())
+  is_mapped_enum = enum.get("generate-mappings", False)
+  enum_name = f"{enum_name}Mapped" if (is_mapped_enum and not enum_name.endswith("Mapped")) else enum_name
   allow_alias = enum.get("allow-alias", proto_helpers.should_allow_alias(enum))
   nonint_index = 1
 %>\
 enum ${enum_name} {
 %   if allow_alias == True:
   option allow_alias = true;
 %   endif
+%   if is_mapped_enum:
```

Comment:
Are you talking about not adding `_MAPPED_` for `UNSPECIFIED` value of non-SetAttribute API enums? That would be difficult to figure out here, since we just have list of `enums` here and don't know which APIs use these enums. We can figure out that information by iterating over all functions, but that will just make this codegen more unreadable.

Reply:
Since I'm saying I don't want "mapped" to be automatically added to enum names, I wouldn't want `_MAPPED_` automatically added to the value names.

Reply:
Even if we don't add "Mapped" to other enums from `enums.py` as per your other comment, `_MAPPED_` needs to be added here for `SetAttribute` APIs, since they might have both mapped and unmapped enums. And both of them will end up having `NIDMM_INT32_UNSPECIFIED` if I don't add `_MAPPED_` explicitly. And even to do this specifically for `SetAttribute` APIs, I will need to know if the enum corresponds to `SetAttribute` API which will further complicate the logic here (I can probably just do `enum_name.endswith("AttributeValues")`).

Other option would be adding `_MAPPED_` to `enum_value_prefix` attribute on consolidated mapped enum when we generate it during mutation. But that means all other values would get `_MAPPED_` prefix too.